.github/workflows/cxx-build.yml

```diff
@@ -121,27 +119,58 @@ jobs:
           key: deps-node-modules-${{ hashFiles('package-lock.json') }}
 
       - name: Install npm dependencies
-        run: npm ci
+        run: |
+          apt update
+          apt install -y cmake
+          npm install
 
       - name: Generate core SDK source code
         run: |
           .github/workflows/utils.sh generate_cpp_core_sdk_source_code
 
-      - name: Build CXX Core SDK 
+      - name: Build CXX Core SDK
         run: |
+          apt install -y python3-pip python3.11-venv
+          python3 -m venv firebolt-env
+          source firebolt-env/bin/activate
+          pip install gcovr
+
           .github/workflows/utils.sh build_core_cpp_sdk
 
+      - name: Test and Generate coverage report
+        run: .github/workflows/utils.sh generate_core_sdk_coverage_report
+
+      - name: Display coverage report
+        uses: irongut/CodeCoverageSummary@v1.3.0
+        with:
+          filename: coverage/coverage.cobertura.xml
+          badge: true
+          fail_below_min: true
+          format: markdown
+          hide_branch_rate: false
+          hide_complexity: true
+          indicators: true
+          output: both
+          thresholds: '10 20'
+      
+      - name: Add coverage PR comment
+        uses: marocchino/sticky-pull-request-comment@v2
+        with:
+          recreate: true
+          path: code-coverage-results.md
+
+
   manage_sdk:
     name: Build Manage SDK
     needs: [thunder, openrpc]
     runs-on: ubuntu-latest
     container:
-      image: kevinshahfws/node-c:3.0
+      image: node:latest
```

Comment:
Can we manually specify whatever the latest version is here? It wouldn't be the first time that "latest" changed to something that broke a pipeline.

Reply:
fixed